src/index.jsx

```diff
@@ -6,10 +6,14 @@ import App from './App';
 import AdminDashboard from './pages/AdminDashboard/AdminDashboard';
 
 import Map from './components/Map/Map';
+import BoxApproval from './components/BoxApproval/BoxApproval';
 
 ReactDOM.render(
   <React.StrictMode>
     <Router>
+      <div id="box-approval-overlay-container">
```

Comment:
Could we remove the box approval component from here? Currently, the box approval component is loading on all pages. For this case, we actually want to add the Map component to the AdminDashboard component instead of adding the box approval component to the map and we only want to shorten the map's width when it is on the AdminDashboard. If you look at the Figma sheet, you'll notice that the admin dashboard and general dashboard are different and that the box approval component is only on the admin dashboard. Let me know if you need more clarification on this :)